src/Components/Calculator.js

```diff
@@ -1,42 +1,84 @@
-import React from "react";
+import React, { useState } from "react";
 import "./Calculator.css";
 
-class Calculator extends React.Component {
+const Calculator = () => {
```

Comment:
- [ ] The Calculator component should import the calculate.js file from the logic directory